index.bs

```diff
@@ -155,20 +155,23 @@ Note, the above algorithm:
     it parses `Accept-CH` and adds the results to the environment setting object's client hints set as well as the Accept-CH cache.
     </div>
 
-<dfn>Accept-CH state</dfn> (`http-equiv="accept-ch"`) {#accept-ch-state-algo}
+<dfn>Accept-CH state</dfn> (`name="accept-ch"`) {#accept-ch-state-algo}
 --------
 
-Note: This pragma *appends* [=client hints token=]s to the [=environment settings object=]'s [=environment settings object/client hints set=]. It *does not* add those hints to the [=Accept-CH cache=].
+Note: This metadata *appends* [=client hints token=]s to the [=environment settings object=]'s [=environment settings object/client hints set=]. It *does not* add those hints to the [=Accept-CH cache=].
 
 1. If the <{meta}> element is not a child of a <{head}> element, then return.
 2. If the <{meta}> element has no <{meta/content}> attribute, or if that attribute's value is the empty string, then return.
-3. Let |settingsObject| be the <{meta}> element's [=relevant settings object=].
-4. If the result of executing [$Is an environment settings object contextually secure?$] on |settingsObject| is `"Not Secure"`, then return.
-5. Let |browsingContext| be |settingsObject|'s  [=environment settings object/global object=]'s [=Window/browsing context=].
-6. If the [=top-level browsing context=] does not equal |browsingContext|, abort these steps.
-7. Let |acceptCHValue| be the <{meta}> element's <{meta/content}> attribute's value.
-8. Parse |acceptCHValue| according to the [=Accept-CH=] header parsing rules, as a [=field-name=].
-     [=set/Append=] each parsed [=client hints token=] to |settingsObject|'s [=environment settings object/client hints set=].
+3. If the <{meta}> element is below any <{link}> or <{script}> element, then return.
+4. Let |settingsObject| be the <{meta}> element's [=relevant settings object=].
+5. If the result of executing [$Is an environment settings object contextually secure?$] on |settingsObject| is `"Not Secure"`, then return.
+6. Let |browsingContext| be |settingsObject|'s  [=environment settings object/global object=]'s [=Window/browsing context=].
+7. If the [=top-level browsing context=] does not equal |browsingContext|, abort these steps.
+8. Let |acceptCHValue| be the <{meta}> element's <{meta/content}> attribute's value.
+9. Parse |acceptCHValue| according to the <a href="https://w3c.github.io/webappsec-permissions-policy/#ascii-serialization">`serialized permission policy`</a> parsing rules
+     while limiting <a href="https://w3c.github.io/webappsec-permissions-policy/#ascii-serialization">`feature-identifier`</a>s to [=client hints token=]s and disallowing 
+     <a href="https://w3c.github.io/webappsec-permissions-policy/#ascii-serialization">`allow-list-value`</a> to be `*`. [=set/Append=] each parsed [=client hints token=]
```

Comment:
s/client hints token/feature identifier/?

Reply:
Also, the `allow-list-value` should impact the relevant permissions-policy, right? Otherwise, this will not impact 3Ps

Reply:
Also a nit: can you try to use refs here instead of direct links?